src/supernova/mod.rs

```diff
@@ -486,12 +499,12 @@ where
         pp.digest(),
         G2::Scalar::ZERO,
         z0_secondary,
-        None,
-        None,
-        Some(&u_primary),
-        None,
-        None,
-        G2::Scalar::from(circuit_index as u64),
+        None,                                   // zi = None
```

Comment:
Nit: weird spacing

Reply:
Added an auxiliary variable to make `rustfmt` play nice.